packages/core/forms/src/index.ts

```diff
@@ -1,11 +1,13 @@
 import type { App } from 'vue'
 import VueFormGenerator from './generator/FormGenerator.vue'
 import * as sharedForms from './forms'
+import attributesDirective from './directives'
 
 // Export Vue plugin as the default
 export default {
   install: (app: App): void => {
     app.component('VueFormGenerator', VueFormGenerator)
+    app.directive('attributes', attributesDirective)
```

Comment:
question: we're making this available in the entire host app; is that desired?

Reply:
It might be better if we could prepend prefixes like `forms` and rename them to a more self-explaining name like: `v-forms-bind-attrs`